tests/functional/raw-node/test/routes/routeBackbeat.js

```diff
@@ -1513,13 +1514,16 @@ describeSkipIfAWS('backbeat routes', () => {
             done();
         }));
```

Comment:
should add tests to verify versioning is not checked when `x-scal-versioning-enabled` is not defined or false.